.gitignore

```diff
@@ -1,3 +1 @@
 .env
-config/config.yaml
```

Comment:
This was ignored to avoid sending sensitive data to there repo. There is file examples on the repo but if it's not clear we should see how to clarify it.